src/porepy/compositional/compositional_mixins.py

```diff
@@ -1095,25 +1129,37 @@ def create_mixture(self) -> None:
 
         self.set_components_in_phases(components, phases)
 
-        self.fluid_mixture = FluidMixture(components, phases)
+        self.fluid = Fluid(components, phases)
 
     def get_components(self) -> list[Component]:
```

Comment:
This method has now a default-behavior, creating a one-component fluid based on the fluid constants passed by the user. Important to mention is that the simulation set-up does not change for the standard 1-phase, 1-component case.
Users still use and pass a FluidConstants instance in params.

Reply:
Can be resolved for my sake.